.gitignore

```diff
@@ -374,6 +374,8 @@ src/Hng.Web/internal-nlog.txt
 
 # AppSettings
 appsettings.*.json
+appsettings.json
+src/Hng.Web/appsettings.json
```

Comment:
Remove this from gitignore. Leave it as it is. You can create another file and pass the values through it, but not this app settings file.